lib/msf/core/post/windows/registry.rb

```diff
@@ -10,16 +10,19 @@ module Registry
 
   include Msf::Post::Windows::CliParse
 
+  REGISTRY_VIEW_NATIVE = 0
+  REGISTRY_VIEW_32_BIT = 1
+  REGISTRY_VIEW_64_BIT = 2
```

Comment:
Could you add some comments here to indicate what each view gets us?  Also, since the default is now ```REGISTRY_VIEW_NATIVE```, how does that differ from what is currently shipping?